ui/shared/ad/CustomAdBanner.tsx

```diff
@@ -0,0 +1,61 @@
+import { Flex, chakra, Tooltip, Image } from '@chakra-ui/react';
+import { useQuery } from '@tanstack/react-query';
+import React, { useState, useEffect } from 'react';
+
+import type { AdCustomConfig } from 'types/client/ad';
+
+import config from 'configs/app';
+import type { ResourceError } from 'lib/api/resources';
+import useFetch from 'lib/hooks/useFetch';
+
+import useIsMobile from '../../../lib/hooks/useIsMobile';
```

Comment:
better to use aliased import - `lib/hooks/useIsMobile`

Reply:
ok